pkgs/os-specific/darwin/apple-source-releases/network_cmds/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, appleDerivation, xcbuild, openssl, Librpcsvc, xnu, libpcap, developer_cmds }:
+
+appleDerivation rec {
+  buildInputs = [ xcbuild openssl xnu Librpcsvc libpcap developer_cmds ];
+
+  NIX_CFLAGS_COMPILE = " -I./unbound -I${xnu}/Library/Frameworks/System.framework/Headers/ -D__APPLE_USE_RFC_2292 -DIPV6_DONTFRAG=14 -DINET6";
+
+  patchPhase = ''
+    # disable spray, rtadvd
```

Comment:
Ideally just like pkill I'd put a little note explaining why these need to be disabled, so someone can know whether to try to enable them in future.